package.json

```diff
@@ -1752,8 +1757,19 @@
                 }
             },
             {
-                "command": "aws.credential.profile.create",
-                "title": "%AWS.command.credential.profile.create%",
+                "command": "aws.credentials.profile.create",
+                "title": "%AWS.command.credentials.profile.create%",
+                "category": "%AWS.title%",
+                "enablement": "!isCloud9",
```

Comment:
Does this prevent the command showing in the command palette? Let's enable it in Cloud9 to start with.

Reply:
No, I think Cloud9 hasn't implemented "enablement". But I'll remove it anyway.